proposal/new-js-api.changes.md

```diff
@@ -0,0 +1,9 @@
+## Draft 1.1
```

Comment:
This should increment the major version, since it affects the intended semantics of the proposal.